assets/app/scripts/controllers/overview.js

```diff
@@ -132,20 +132,23 @@ angular.module('openshiftConsole')
     // Filter out monopods we know we don't want to see
     function showMonopod(pod) {
       // Hide pods in the Succeeded or Terminated phase since these are run once pods
-      // that are done
+      // that are done.
       if (pod.status.phase == 'Succeeded' || pod.status.phase == 'Terminated') {
```

Comment:
Wonder if we should include "Failed" in the excluded list. Otherwise failed pods show up forever. Maybe spawn an issue/PR for that